electron-react/src/components/farm/card/FarmCardCoinbaseRewards.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useMemo } from 'react';
+import { useAsync } from 'react-use';
+import { Trans } from '@lingui/macro';
+import { useSelector } from 'react-redux';
+import type { RootState } from '../../../modules/rootReducer';
+import FarmCard from './FarmCard';
+import computeStatistics from '../../../util/computeStatistics';
+import { mojo_to_chia } from '../../../util/chia';
+
+export default function FarmCardCoinbaseRewards() {
+  const wallets = useSelector((state: RootState) => state.wallet_state.wallets);
+  const { loading, value } = useAsync(() => computeStatistics(wallets), [
+    wallets,
+  ]);
+
+  const coinbaseRewards = useMemo((): number => mojo_to_chia(value?.coinbaseRewards), [value?.coinbaseRewards]);
+
+  return (
+    <FarmCard
+      title={
+        <Trans id="FarmCardCoinbaseRewards.title">XCH Framing Rewards</Trans>
```

Comment:
We need to remain TXCH unless showing mainnet balances

Reply:
I already changed it